src/DatePicker/__tests__/DatePicker.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import createWithTheme from '../../../utils/createWithTheme';
+import DatePicker from '../DatePicker';
+
+describe('<DatePicker />', () => {
+  it('properly renders a DatePicker', () => {
+    const tree = createWithTheme(<DatePicker />);
+
+    expect(tree).toMatchSnapshot();
+  });
+
+  describe('props', () => {
+    it('passes props along to the StyledDatePicker', () => {
+      const datePicker = shallow(
+        <DatePicker
+          foo="bar"
+          navbarElement="foo"
+          showOutsieDays={false}
+          weekdaysShort={['A', 'N', 'T', 'H', 'O', 'N', 'Y']}
```

Comment:
Is this expected to be passed to `StyleDatePicker` too? Would it make sense to add it to the assertion below if so? Otherwise it's not clear why this, and the other props are being passed in.

Reply:
This was left over from when I was forcing certain values. I'll remove it.